### ISSUE TITLE
Coloring the Connecting branches

### ISSUE THREAD

**Issue by chaunceychason** [schlegelp/tanglegram#3]
Would it be possible to implement the color functionality contained in  the original R code so that the middle panel isnt just grey? I have tons of items in my dendrogram, so currently the connecting bars are all running together as a single grey bar. Being able to change the color on those objects would be super helpful

**Comment by schlegelp** [comment]
Could you clarify please? I see three options to colourise off the top of my head:
1. Give connecting lines have the same colour as the leafs in the dendrogram. What if the left and the right leaf have different colours?
2. Assign custom colours to each connecting line?
3. Map entanglement to RGB value instead of shades of grey.

**Comment by chaunceychason** [comment]
Thanks for the reply. I noticed the function doesnt return a plot object to
modify. I could easily manually assign the colors if the API allows it. Is
there a fig or plt instance for each subplot?

**Comment by schlegelp** [comment]
Yes you're right. The function returns a matplotlib figure. It has three axes which you can get with `fig.get_axes()`. The first two are the left and right dendrogram respectively, the last one is the centre lines. Have a go at it and let me know if you figure out a way to colourise them that makes sense to you.

I'll try to implement a way to do this natively.
